client/jetpack-cloud/sections/partner-portal/issue-license-v2/review-licenses/license-info.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslate } from 'i18n-calypso';
+import getProductIcon from 'calypso/my-sites/plans/jetpack-plans/product-store/utils/get-product-icon';
+import { useLicenseLightboxData } from '../../license-lightbox/hooks/use-license-lightbox-data';
+import type { SelectedLicenseProp } from '../types';
+
+export default function LicenseInfo( { product }: { product: SelectedLicenseProp } ) {
+	const translate = useTranslate();
+
+	const { title, product: productInfo } = useLicenseLightboxData( product );
+
+	if ( ! productInfo ) {
+		return null;
+	}
+
+	return (
+		<div className="review-licenses__selected-license">
+			<div className="review-licenses__details">
```

Comment:
[non-blocker] I suggest that you add the prefix "review-licenses__selected-license" to the child elements such as "review-licenses__license-details" and "review-licenses__license-info", provided that these child elements are not used elsewhere. This will help to clarify the relationship between the container and its child elements and make it easier for readers to understand if these are standalone elements.